File: Loadshop.DomainServices/Loadshop/Services/Data/Feedback/QuestionData.cs
using System.Collections.Generic;

namespace Loadshop.DomainServices.Loadshop.Services.Data
{
    public class QuestionData
    {
        public FeedbackQuestionCodeEnum FeedbackQuestionCode { get; set; }

        public int? QuestionId { get; set; }
        public string ApplicationCode { get; set; }
        public string Category { get; set; }
        public string QuestionText { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
        public IList<QuestionReasonData> QuestionReasons { get; set; }
        public string UpdateUser { get; set; }
    }
}

File: Loadshop.Web.UI/src/app/shared/models/question-data.ts
import { QuestionReasonData } from './question-reason-data';
import { FeedbackQuestionEnum } from './feedback-question-enum';

export interface QuestionData {
    feedbackQuestionCode: FeedbackQuestionEnum;
    questionId: number;
    applicationCode: string;
    category: string;
    questionText: string;
    description: string;
    isActive: boolean;
    questionReasons: QuestionReasonData[];
    updateUser: string;
}
